LICENSE_THIRDPARTY

```diff
@@ -0,0 +1,27 @@
+When building POCL for macOSX, the files lib/CL/devices/pthread/pthread_barrier.*
+are used which are licensed as below.
+
+Copyright (c) 2015, Aleksey Demakov
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```

Comment:
How does this differ from the MIT license most of the PoCL is licensed with? I don't see the difference, but IANAL.

Reply:
It is the [2-clause BSD license](https://github.com/ademakov/DarwinPthreadBarrier/blob/master/LICENSE). I guess having some attribution here wouldn't hurt?